docs: add note referencing `kind load image-archive --help`

AnjaliMishra1st opened the pull request:
Adds a short note pointing users to the CLI help output for additional flags, as suggested in issue #4051.

AnjaliMishra1st committed:
docs: add note referencing `kind load image-archive --help`

Adds a short note pointing users to the CLI help output for additional flags, as suggested in issue #4051.

AnjaliMishra1st commented:
Hi @BenTheElder and @aojea,
This is a small documentation update based on the guidance in issue #4051.  
Whenever you have time, could you please review and trigger /ok-to-test?
Thanks!

AnjaliMishra1st committed:
Update site/content/docs/user/quick-start.md

Co-authored-by: Sean McGinnis <sean.mcginnis@gmail.com>

AnjaliMishra1st committed:
Enhance quick-start guide with image loading example

Added example workflow for loading image archives.

BenTheElder commented on site/content/docs/user/quick-start.md:
```diff
@@ -218,25 +218,23 @@ context name `kind` and delete that cluster.
 Additionally, image archives can be loaded with:
 `kind load image-archive /my-image-archive.tar`
 
+For additional usage details and available flags, you can run:
```
Maybe instead we should document somewhere that `kind help` exists.

That recursively documents `--help` and using it for subcommands. I don't think we want to add an entry like this to the docs for every command.

AnjaliMishra1st committed:
Enhance quick-start.md with command help details

Updated documentation to include command help options for 'kind'.

AnjaliMishra1st commented:
Hi @BenTheElder,  
I have updated the PR according to your suggestion and added a general help
section instead of referencing --help for the specific command.

Please take a look when convenient. Thank you!

stmcginnis commented on site/content/docs/user/quick-start.md:
````diff
@@ -218,26 +218,34 @@ context name `kind` and delete that cluster.
-```bash
-kind load docker-image my-app:latest --name test-cluster
-```
 
````
nit: extra line not needed, so remove this if you do another update.

stmcginnis commented on site/content/docs/user/quick-start.md:
```diff
@@ -218,26 +218,34 @@ context name `kind` and delete that cluster.
 Additionally, image archives can be loaded with:
 `kind load image-archive /my-image-archive.tar`
 
+
```
Same here. If you use a markdown linter, they typically only want one blank line between elements.

stmcginnis reviewed (commented):
I do like this better than just referencing `--help`.

Now my concern is placement. It kind of feels like it is just dropped in the middle of a different context. I don't know the best place for it, but what do folks think about it being a subsection at the end of the Installation section? That sort of seems logical to me because that section is about getting started with the CLI itself, then the rest of the sections are about performing specific tasks with it.

AnjaliMishra1st committed:
Update quick-start.md

AnjaliMishra1st commented:
Hi @stmcginnis @BenTheElder,

I’ve updated the placement as suggested and fixed the markdown nits.  
All checks are passing now.
Whenever convenient, could you please re-add /lgtm and /approve?
Thanks!

BenTheElder reviewed (commented):
/approve
